lib/pacemaker/pcmk_sched_native.c

```diff
@@ -552,10 +552,7 @@ RecurringOp(pe_resource_t * rsc, pe_action_t * start, pe_node_t * node,
     }
 
     if (rsc->next_role == RSC_ROLE_PROMOTED) {
-        char *running_promoted = pcmk__itoa(PCMK_OCF_RUNNING_PROMOTED);
-
-        add_hash_param(mon->meta, XML_ATTR_TE_TARGET_RC, running_promoted);
-        free(running_promoted);
+        pe__add_action_expected_result(mon, CRM_EX_PROMOTED);
```

Comment:
Why the switch from PCMK_OCF_ to CRM_EX_ throughout?

Reply:
I should have done that in a separate commit ... The PCMK_OCF_ codes are a subset of CRM_EX_, and I decided we didn't need two enums covering the same ground. I don't think any code uses PCMK_OCF_ anymore, so we could deprecate those, except the enum type is still used in public APIs.